.gitignore

```diff
@@ -0,0 +1 @@
+connect-cli
```

Comment:
This looks specific to your environment. There's already a .gitignore in `connect`. I would suggest simply removing this file.

Reply:
You are right, not my best PR :-)